### PR TITLE
Remove binaural decoders that do not compile on Windows (VS 2017) or on Embedded Linux

### PR DESCRIPTION
**Pull request by florian-grond**
Hi everyone, I removed the binaural decoders that do not compile on the RPi, so that we can set the compile flag for the HOA plugis to on. 
We plan on making the Decoders available in a different repository.
Binaural decoding is still possible over the Lebedev grids

### CODE REVIEW

File: source/CMakeLists.txt [hunk 1]
```diff
@@ -175,6 +175,17 @@ if (HOA_UGENS)
...
+#    FILE(GLOB_RECURSE HOAUGensSources HOAUGens/*cpp)
+#    FOREACH(item ${HOAUGensSources})
+#        IF(${item} MATCHES "foo/main.c")
+#          LIST(REMOVE_ITEM files ${item})
+#        ENDIF(${item} MATCHES "foo/main.c")
+#    ENDFOREACH(item)
+#    MESSAGE(STATUS "${files}")
+#endif()
+
+
```
Comment (patrickdupuis) [inline source/CMakeLists.txt]: Please remove this commented out block of code.

Comment (florian-grond) [reply source/CMakeLists.txt]: done

File: source/CMakeLists.txt [hunk 2]
```diff
@@ -175,6 +175,8 @@ if (HOA_UGENS)
     endif()
 endif()
 
+
+
```
Comment (patrickdupuis) [inline source/CMakeLists.txt]: For the sake of cleanliness, you should also remove these new empty lines.

Comment (florian-grond) [reply source/CMakeLists.txt]: removed

**Comment by florian-grond** [comment]
@patrickdupuis  when it passes all the tests, maybe we can then set the compile flag for the HOA_UGENS to ON?

**Comment by patrickdupuis** [comment]
Let's make the necessary changes to cmake in this PR as well. The CI will test Windows for us once the commits are in. If it all looks good then I can test on Rpi.

**Comment by florian-grond** [comment]
would you mind to make the cmake changes?

**Comment by patrickdupuis** [comment]
yes

**Comment by patrickdupuis** [comment]
Redoing this as #206

**Pull request closed without merging**